cloud-bucket-upload-page/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+
+    <img class="logo" src="../CoVision logo small transp.png" />
+    <h1>Rapid Test Image Upload</h1>
+    <form action="https://storage.googleapis.com/covisionapp_image_upload" method="post" enctype="multipart/form-data">
+        <input type="hidden" name="key" value={Math.random().toString().substring(2)}>
```

Comment:
this would only work in React apps; here we'd need to use JS to update the value of this field on page load